app/controllers/decidim/anonymous_codes/admin/code_groups_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      class CodeGroupsController < ApplicationController
+        helper_method :groups
+
+        def index; end
```

Comment:
```suggestion
        def index
          @groups = paginate(groups)
        end
```
Instead of using the helper groups directly (which maybe we don't need anymore as a helper), let's use a class var that will be available in the view to iterate, paginated

Reply:
Done